data/geography_matrix.csv

```diff
@@ -0,0 +1,18 @@
+row_number,geographic_level,code_field,name_field,code_field_secondary
```

Comment:
Oooh. I like this suggestion a lot. Though, I will politely tell you to sod off for this PR at least and suggest we do this at a later point.

I've raised #127 for this, as it would be nice to move to this. Though when I did the rewrite in this PR I looked for all references to the geography_matrix, if you look through for any references to geographic locations / columns there's a whole raft more of hard coded stuff, which is partly what chipped away at my attempts to guard against it as I realised I was trying to hold back the floodgates while a river was happily also running past me!